resources/prefabs/creatures/herbivore.ron

```diff
@@ -31,6 +31,9 @@ Prefab (
                         max: 100.0,
                         value: 100.0,
                     ),
+                    nutrition: (
```

Comment:
Could you add nutrition also to plants and herbivores? You can look up the initial values here: https://community.amethyst.rs/t/evoli-mvp-specification/487 (under `Creature attributes`)
Herbivores can't be eaten yet, but we can add it anyway :)

Reply:
Added this in https://github.com/amethyst/evoli/pull/69/commits/37f8defcfce7ac4c63764f711dbaf3ccda44b18f

Reply:
Great. thanks :)